### PR TITLE
NFT-838: New vault indicator

### PR DESCRIPTION
**Pull request by adamgobes**
Screen Shot 2023-02-02 at 12 12 32 PM

### CODE REVIEW

File: components/Controllers/Loans/VaultHealth.tsx [hunk 1]
```diff
@@ -21,22 +21,25 @@ export function VaultHealth({ ltv }: VaultHealthProps) {
...
       <TooltipReference {...healthTooltip}>
-        <span
+        <div
           className={
             ratio > 0.5 ? styles['indicator-danger'] : styles.indicator
           }>
-          {indicator}
-        </span>
+          {indicator.map((char, i) => (
+            <p key={i}>{char}</p>
```
Comment (cnasc) [inline components/Controllers/Loans/VaultHealth.tsx:40]: what's the reason for wrapping as a `p`?

Comment (adamgobes) [reply components/Controllers/Loans/VaultHealth.tsx:40]: guess i could have done a span, is that more semantic?

Comment (adamgobes) [reply components/Controllers/Loans/VaultHealth.tsx:40]: just so i'd have an element to select via css to add margin between them

**Comment by jringenberg** [comment]
@adamgobes When I try to view the preview I get a bunch of errors and the page doesn't load so I can't inspect, but are you assigning a specific font to those ░░▓▓?  I did some testing and I notice that if the font is Arial they are rendered thinner and a little more seamlessly from one character to the next. I think Arial is what Chrome uses. It would be worth exploring these characters as Arial, in which case we wouldn't get those gaps between the characters (though that may be caused by letter-spacing, in which case we should take that off). With a narrower character rendering we could also get more, 10 instead of 6? That might feel more exact and therefore more like a bar chart.

File: components/Controllers/Loans/VaultHealth.tsx (hunk 1 above)
Comment (cnasc) [reply components/Controllers/Loans/VaultHealth.tsx:40]: span would be preferable but not critical

**Comment by wilsoncusack** [comment]
@adamgobes mahbe push to rosy so joe can better see?

**Comment by adamgobes** [comment]
@jringenberg pushed to rosy

**Review by cnasc — APPROVED** [review]
:shipit:

**Pull request merged**